winediscordipcbridge-steam.sh

```diff
@@ -23,6 +23,7 @@ DISCORD_IPC_PATHS=(
     "$TEMP_PATH/app/com.discordapp.Discord"
     "$TEMP_PATH/snap.discord-canary"
     "$TEMP_PATH/snap.discord"
+    "run/user/$UID"
```

Comment:
Is this supposed to be a relative path or is it missing a leading slash?

Reply:
Missing Slash, resolved